ocp_resources/resource_quota.py

```diff
@@ -0,0 +1,27 @@
+# API reference: https://docs.openshift.com/container-platform/4.13/applications/quotas/quotas-setting-per-project.html
+
+from ocp_resources.resource import NamespacedResource
+
+
+class ResourceQuota(NamespacedResource):
+    api_version = NamespacedResource.ApiVersion.V1
+
+    def __init__(
+        self,
+        hard=None,
```

Comment:
according to the [api](https://kubernetes.io/docs/reference/kubernetes-api/policy-resources/resource-quota-v1/#ResourceQuotaSpec), the spec also has `scopeSelector` and `scopes`, any reason why not to add those?

Reply:
I didn't have a goal to add the support for all possible fields.. I was thinking - they can be added later when (or rather, if) someone needs them